app/src/main/res/layout/activity_multimeter.xml

```diff
@@ -1,487 +1,627 @@
 
         <TextView
-            android:id="@+id/tv_control_read1"
-            android:layout_width="match_parent"
-            android:layout_height="@dimen/multimeter_display_height"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
             android:layout_gravity="center"
-            android:layout_marginLeft="@dimen/multimeter_display_margin"
-            android:layout_marginRight="@dimen/multimeter_display_margin"
-            android:layout_marginTop="@dimen/multimeter_display_margin_top"
-            android:background="@drawable/tv_border"
+            android:layout_weight="1"
             android:gravity="center"
+            android:text="@string/id1"
+            android:textSize="@dimen/control_textsize_tiny"
             android:textStyle="bold" />
 
         <TextView
-            android:id="@+id/quantity"
             android:layout_width="wrap_content"
-            android:layout_height="@dimen/multimeter_display_text_margin"
-            android:layout_alignParentRight="true"
-            android:layout_marginRight="@dimen/multimeter_layout_margin"
-            android:layout_marginTop="@dimen/multimeter_display_text_margin"
-            android:background="@android:color/white"
-            android:fontFamily="sans-serif"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:layout_marginLeft="@dimen/multimeter_layout_margin"
+            android:layout_marginStart="@dimen/multimeter_layout_margin"
+            android:layout_weight="1"
             android:gravity="center"
-            android:textSize="@dimen/multimeter_diplay_textsize" />
+            android:text="@string/id4"
+            android:textSize="@dimen/control_textsize_tiny"
+            android:textStyle="bold" />
+
+    </LinearLayout>
 
-        <View
-            android:layout_width="match_parent"
-            android:layout_height="@dimen/multimeter_line_width"
-            android:layout_marginLeft="@dimen/multimeter_display_margin"
-            android:layout_marginRight="@dimen/multimeter_display_margin"
-            android:layout_marginTop="@dimen/multimeter_displayline_topmargin"
-            android:background="@android:color/darker_gray" />
+    <LinearLayout
+        android:id="@+id/linearLayout14"
+        android:layout_width="@dimen/multimeter_ll14_width"
+        android:layout_height="@dimen/multimeter_ll14_height"
+        android:layout_gravity="center"
+        android:layout_marginEnd="@dimen/multimeter_ll14_margin_right"
+        android:layout_marginLeft="@dimen/multimeter_ll14_margin_left"
+        android:layout_marginRight="@dimen/multimeter_ll14_margin_right"
+        android:layout_marginStart="@dimen/multimeter_ll14_margin_left"
+        android:layout_marginTop="@dimen/multimeter_ll14_margin_top"
+        app:layout_constraintEnd_toEndOf="@+id/knob_line6"
+        app:layout_constraintHorizontal_bias="0.42"
```

Comment:
This can too can't it. There are other places as well.

Reply:
This are hardcoded because these dimensions are particularly for app.

Reply:
It's fine then :+1: